app/controllers/alchemy/admin/resources_controller.rb

```diff
@@ -90,8 +92,54 @@ def resource_handler
         @_resource_handler ||= Alchemy::Resource.new(controller_path, alchemy_module)
       end
 
+      def resource_has_filters
```

Comment:
I am moving `resource_handler` to the list of protected methods, but leaving `resource_handler` where it is because it is used a lot in view partials and in `resources_helper`.

Reply:
I just realised this is not possible as `resource_has_filters` is used in the view partials as well. I reverted the change.